crates/payload/validator/src/lib.rs

```diff
@@ -112,15 +113,12 @@ impl<ChainSpec: EthereumHardforks> ExecutionPayloadValidator<ChainSpec> {
     pub fn ensure_well_formed_payload(
         &self,
         payload: ExecutionPayload,
-        cancun_fields: MaybeCancunPayloadFields,
-        execution_requests: Option<Requests>,
+        sidecar: ExecutionPayloadSidecar,
     ) -> Result<SealedBlock, PayloadError> {
         let expected_hash = payload.block_hash();
 
         // First parse the block
-        let sealed_block =
-            try_into_block(payload, cancun_fields.parent_beacon_block_root(), execution_requests)?
-                .seal_slow();
+        let sealed_block = try_into_block(payload, sidecar.clone())?.seal_slow();
```

Comment:
I think this is relatively cheap but we could likely use &sidecar here?

Reply:
yeah true